docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,608 @@
+  - is there at least one active signing group remaining? +
+
+| Relay request
+| If a relay request is submitted before the previous relay request finished
+  being processed, the new request should be rejected (transaction should be
+  reverted).
+
+| Relay request
+| If the relay request is deemed valid, then: +
+  - the fee charged from the requester should be deposited in the rewards pool
+  (the T balance of the random beacon contract) +
+  - signing groups that reached the expiration threshold should get expired +
+  - the last relay entry should be used by the beacon to select a signing
+  group from the set of active groups (by hashing the relay entry and applying
+  `modulo no_of_active_groups` operation on the result) +
+  - the random beacon contract should emit an event informing about the group
+  selected to sign the entry +
+  - the beacon should start the period of signature submission eligibility +
+  - the selected signing group should start off-chain signature generation (upon
+  noticing event sent by the beacon). +
+  - the sortition pool should get locked as a preparation for creation of a new
+  signing group (only sometimes, every N-th relay request).
+
+| Relay request
+| If the relay request is deemed invalid, then: +
+  - the transaction should be reverted.
+
+| Relay request - signature generation
+| Once the active group is selected as a signing group, it is tasked with the
+  generation of a BLS threshold signature (based on the last relay entry and
+  the data provided by at least `blsThreshold=(group_size/2) + 1` members of the
+  signing group). This signature will be used as a proposed new relay entry.
+
+ | Relay request - relay entry submission
+ | Anyone can send a transaction submitting a new relay entry.
+
+ | Relay request - relay entry submission
+ | Off-chain clients are expected to agree and follow the following order to
+   minimize the gas cost and distribute costs: +
+   - at the beginning only one member, with index `new_entry % group_size`,
+  should be eligible to submit a new relay entry +
+  - after fixed amount of blocks another member, with index `(new_entry %
+  group_size ) + 1`, should become eligible (previous operator remains eligible) +
+  - next members should become eligible after each fixed amount of blocks.
+
+| Relay request - punishing inactive operators
+| It should be possible for a member of the group to send a notication informing
+  about group members notoriously ignoring their duty in relay entry (what will
+  be considered as ignoring the duties and how will it be checked should depend
+  on the off-chain algorithm - suggested mechanism: heartbeats).
+  
+| Relay request - punishing inactive operators
+| Upon receiving the transaction notifying about the misbehaving operators, the
+  random beacon should determine its validity.
+  
+| Relay request - punishing inactive operators
+  When determining validity of the transaction notifying about the misbehaving
```

Comment:
We should probably have `|` before `When determining validity of the transaction notifying about the misbehaving`. It probably causes the `Area` and `UAC` columns mixed up when we view [this document](https://github.com/keep-network/keep-core/blob/random-beacon-uacs/docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc) with a browser.

Reply:
Right, fixed in 2866f5ea524741025d92299c89b1bd2441274b52.